modin/pandas/test/test_io.py

```diff
@@ -353,16 +353,7 @@ def test_read_csv_parsing_1(
     @pytest.mark.parametrize("skiprows", [2, lambda x: x % 2])
     @pytest.mark.parametrize("skipfooter", [0, 10])
     @pytest.mark.parametrize("nrows", [35, None])
-    @pytest.mark.parametrize(
-        "names",
-        [
-            pytest.param(
-                ["c1", "c2", "c3", "c4"],
-                marks=pytest.mark.xfail(reason="Excluded because of the issue #2845"),
-            ),
-            None,
-        ],
-    )
+    @pytest.mark.parametrize("names", [["c1", "c2", "c3", "c4"], None])
```

Comment:
The reason of disabling these tests is that if NPartitions is equal or more than 43, the tests will be failing. Since we have NPartitions.put(4) at the top of this file the tests are passing. Reopening conforming issue #2845 .

Reply:
I see, it was auto-closed because I flagged the issue as resolved after I removed the xfail. Since we now have `xfail: strict` we cannot have any passing `xfail` tests.

Reply:
Yes, I saw the change. Also, look at the comment below on 2846.